tests/fips/mozilla_nss/firefox_nss.pm

```diff
@@ -80,7 +83,31 @@ sub run {
 
     # Close Firefox
     quit_firefox;
-    assert_screen("generic-desktop", 20);
+
+    # Add more time for aarch64 due to worker performance problem
+    wait_still_screen 60 if is_aarch64;
```

Comment:
If just want to add more wait time for aarch64, I suggest to do as below:

my $waittime = 60;
$waittime += 60 if is_aarch64;
assert_screen("generic-desktop", $waittime);

WDYT?

Reply:
It looks better than just adding the fixed value, I will change it, thanks a lot.

Reply:
Done :)